libnmstate/nm/ipv6.py

```diff
@@ -113,7 +113,16 @@ def create_setting(config, base_con_profile):
         setting_ip = nmclient.NM.SettingIP6Config.new()
 
     if not config or not config.get(InterfaceIPv6.ENABLED):
-        setting_ip.props.method = nmclient.NM.SETTING_IP6_CONFIG_METHOD_IGNORE
+        if nm_support_disable_ipv6():
```

Comment:
Perhaps it will be more appropriate not to ask and just try:
```
try:
    method = nmclient.NM.SETTING_IP6_CONFIG_METHOD_DISABLED
except ...:
    method = nmclient.NM.SETTING_IP6_CONFIG_METHOD_IGNORE
setting_ip.props.method = method
```

If you extract the method choice logic in a function, it can be used by the unit test: https://github.com/nmstate/nmstate/pull/447/commits/614f1b395d46d4b53f9a35a40a48d445f5af7cd9#diff-84cb5aa9d50a7a47213e550005c6871bR45